### ISSUE TITLE
Markdown Preview Links to sections Not Opening in Folders

### ISSUE THREAD

**Issue by garu57** [foambubble/foam#920]
- Foam version: 0.17.3
- Platform: Windows
- Issue occur on the [foam template](https://github.com/foambubble/foam-template) repo: Yes
**Summary**
It looks the solution for #787 is not complete, it still fails for links to sections.

Two files in `docs\features` dir.
`aaaaa-test-note.md` and `bbbbb-test-note.md`
`aaaaa-test-note.md` contains three links:
```
[link1](bbbbb-test-note.md)
[[features/bbbbb-test-note]]
[[features/bbbbb-test-note#sec2]]
```
created from autocompletion menu.

**All three** works from raw file, **the first two** works from preview.
**The third** fails from preview as described in #787.

**Comment by riccardoferretti** [comment]
Yup, this is a know bug that seems to be harder than originally thought to be fixed
